membership-common/src/main/scala/com/gu/zuora/soap/Client.scala

```diff
@@ -105,25 +105,9 @@ class Client(
   def parent[P <: Query, C <: Query](child: C, foreignKey: C => String)(implicit reader: readers.Query[P]): Future[P] =
     queryOne[P](parentFilter(child, foreignKey))(reader)
 
-  def children[P <: Query with Identifiable, C <: Query](parent: P)(implicit reader: readers.Query[C]): Future[Seq[C]] =
-    query[C](childFilter(parent))(reader)
-
   def child[P <: Query with Identifiable, C <: Query](parent: P)(implicit reader: readers.Query[C]): Future[C] =
     queryOne[C](childFilter(parent))(reader)
 
-  private val lastPingTimeTask =
-    ScheduledTask[Option[DateTime]]("ZuoraPing", None, 0.seconds, 30.seconds) {
-      val result = authenticatedRequest(Actions.Query("SELECT Id FROM Product", enableLog = false))
```

Comment:
I don't think we are relying on any side effects here e.g. to keep the credentials fresh, but deleting this will stop any intended or unintended side effects of this operation